manager/ui/war/plugins/api-manager/html/api/api-def.html

```diff
@@ -31,20 +31,18 @@
                  apiman-i18n-key="api-def">API Definition</div>
             <div class="apiman-label-faded">
               <p apiman-i18n-key="api-def-explanation">
-                Here you can (optionally) provide definition information about your API.  This
-                information should take the form of a Swagger API definition file.  Adding a 
-                definition file will allow consumers to better understand how to use your API.
-                Simply copy/paste (or drag and drop!) your Swagger API definition into the text
-                area below.
+                It is optional to provide definition information about your API in the form of a Swagger API definition

```

Comment:
Optionally, a Swagger or OpenAPI definition about your API can be provided. Adding [...]


(do we support OpenAPI now 🤔, if not, miss out that bit?).